crates/holochain/src/conductor/api/api_external/admin_interface.rs

```diff
@@ -172,6 +172,26 @@ impl AdminInterfaceApi {
                     .await?;
                 Ok(AdminResponse::AgentPubKeyGenerated(agent_pub_key))
             }
+            RevokeAgentKey(payload) => {
+                let RevokeAgentKeyPayload { agent_key, app_id } = *payload;
+                let results = self
+                    .conductor_handle
+                    .clone()
+                    .revoke_agent_key_for_app(agent_key, app_id)
+                    .await?;
+                // Convert errors to strings
+                let results: Vec<(CellId, String)> = results
```

Comment:
This imitates what `EnableApp` does. It returns a vector of tuples of `(cell, error)` of the cells where deletion failed.

Reply:
I guess that makes sense given that EnableApp does the same thing. Now that I'm looking at it though, even though I wrote it, I don't like how Enable app returns a successful response but with errors included. If you have any inclination of changing this, I'd support it. But it's a small fry.

Reply:
Before matching what `EnableApp` does, I wanted to return a map with key `cell_id` and value `result`, with oks and errs of all cells. It also felt funny to return an `Ok` of the overall call with errors of the failed cells. The alternative is to return an `Err` with some successful cells. That's very similar. Out of the two I prefer how it's done currently,

So if I changed anything now, I'd return maps with all results. Unless you have another suggestion which makes everything just peachy.

Reply:
I don't think there's an easy way to achieve peachiness right now, so it's probably better to not change anything. Err doesn't have any special meaning outside of Rust anyway, which is where this is mostly consumed.